build.gradle

```diff
@@ -173,9 +173,6 @@ task coverage(type: JacocoReport) {
 
 String getRepoSenseVersion() {
     String repoSenseVersion = project.property("version")
-    if (repoSenseVersion.equals("unspecified")) {
```

Comment:
This one is needed for our `surge` preview. Maybe, leave the surge deploy with travis first? And in the meantime, we look for an alternative for `id 'com.palantir.git-version' version '0.11.0'`? Anyway, the deployment won't take up so much time?

Reply:
Upgraded this plugin to the latest version `0.12.3` seems to work now